From 7650349d79a33cfc19c87092992ab978ae3a6a6e Mon Sep 17 00:00:00 2001
From: Chris Zhong <zyw@rock-chips.com>
Date: Tue, 15 Sep 2015 09:29:44 +0800
Subject: [PATCH] ARM: dts: add a alias for rockchip_suspend

Since we need overwrite this rockchip_suspend node in sub dts file,
sometimes. Add a alias for before it.

Change-Id: I6d951d1c0bfff1cde619906eb9f11256d057a9fe
Signed-off-by: Chris Zhong <zyw@rock-chips.com>
---
 arch/arm/boot/dts/rk3036.dtsi         |  2 +-
 arch/arm/boot/dts/rk3126-86v.dts      | 12 ++++++------
 arch/arm/boot/dts/rk3126-sdk.dts      |  3 ++-
 arch/arm/boot/dts/rk3128-86v.dts      |  2 +-
 arch/arm/boot/dts/rk3128-box-rk88.dts |  3 ++-
 arch/arm/boot/dts/rk3128-box.dts      |  3 ++-
 arch/arm/boot/dts/rk3188.dtsi         | 13 +++++++------
 arch/arm/boot/dts/rk3288.dtsi         |  2 +-
 8 files changed, 22 insertions(+), 18 deletions(-)

diff --git a/arch/arm/boot/dts/rk3036.dtsi b/arch/arm/boot/dts/rk3036.dtsi
index c16880880762..c65533362ad7 100755
--- a/arch/arm/boot/dts/rk3036.dtsi
+++ b/arch/arm/boot/dts/rk3036.dtsi
@@ -735,7 +735,7 @@
 		interrupt-names = "vpu_mmu";
 	};
 
-	rockchip_suspend {
+	rockchip_suspend: rockchip_suspend {
 		rockchip,ctrbits = <
 			(0
 			 //|RKPM_CTR_PWR_DMNS
diff --git a/arch/arm/boot/dts/rk3126-86v.dts b/arch/arm/boot/dts/rk3126-86v.dts
index a1820d15b32f..633dc603fc4a 100755
--- a/arch/arm/boot/dts/rk3126-86v.dts
+++ b/arch/arm/boot/dts/rk3126-86v.dts
@@ -24,12 +24,12 @@
 	usb_control {
 		compatible = "rockchip,rk3126-usb-control";
 		//host_drv_gpio = <&gpio2 GPIO_B4 GPIO_ACTIVE_LOW>;
-		//otg_drv_gpio = <&gpio2 GPIO_B6 GPIO_ACTIVE_LOW>; 
+		//otg_drv_gpio = <&gpio2 GPIO_B6 GPIO_ACTIVE_LOW>;
 
 		rockchip,remote_wakeup;
 		rockchip,usb_irq_wakeup;
 		};
-		
+
         wireless-wlan {
         compatible = "wlan-platdata";
 
@@ -50,9 +50,9 @@
         //WIFI,reset_gpio = <&gpio0 GPIO_A2 GPIO_ACTIVE_LOW>;
 
         status = "okay";
-    }; 
-    
-    	rockchip_suspend {
+    };
+
+	rockchip_suspend: rockchip_suspend {
                 rockchip,ctrbits = <
                         (0
                         |RKPM_CTR_PWR_DMNS
@@ -68,7 +68,7 @@
                 >;
                 rockchip,pmic-suspend_gpios = <0>;
         };
-	
+
     chosen {
 	bootargs = "vmalloc=496M cma=4M rockchip_jtag";
        };
diff --git a/arch/arm/boot/dts/rk3126-sdk.dts b/arch/arm/boot/dts/rk3126-sdk.dts
index ff2228d3bcfc..5173e4b44e1c 100755
--- a/arch/arm/boot/dts/rk3126-sdk.dts
+++ b/arch/arm/boot/dts/rk3126-sdk.dts
@@ -39,7 +39,8 @@
 	       
 	       satus = "okay";	
 	 };
-       rockchip_suspend {
+
+	rockchip_suspend: rockchip_suspend {
                 rockchip,ctrbits = <
                         (0
                         |RKPM_CTR_PWR_DMNS
diff --git a/arch/arm/boot/dts/rk3128-86v.dts b/arch/arm/boot/dts/rk3128-86v.dts
index 86edff033785..112f5ddf8bf4 100755
--- a/arch/arm/boot/dts/rk3128-86v.dts
+++ b/arch/arm/boot/dts/rk3128-86v.dts
@@ -69,7 +69,7 @@
 		status = "okay";
 	};
 
-      rockchip_suspend {
+	rockchip_suspend: rockchip_suspend {
                            rockchip,ctrbits = <
                            (0
                            |RKPM_CTR_PWR_DMNS
diff --git a/arch/arm/boot/dts/rk3128-box-rk88.dts b/arch/arm/boot/dts/rk3128-box-rk88.dts
index 4c265eee23c0..3158daa638e4 100755
--- a/arch/arm/boot/dts/rk3128-box-rk88.dts
+++ b/arch/arm/boot/dts/rk3128-box-rk88.dts
@@ -93,7 +93,8 @@
 		/*0 - Normal, 1 - Force Host, 2 - Force Device*/
 		rockchip,usb-mode = <1>;
 	};
-	 rockchip_suspend {
+
+	rockchip_suspend: rockchip_suspend {
                 rockchip,ctrbits = <
                         (0
                         |RKPM_CTR_PWR_DMNS
diff --git a/arch/arm/boot/dts/rk3128-box.dts b/arch/arm/boot/dts/rk3128-box.dts
index c42731ffe9da..bcc676e9a6e8 100755
--- a/arch/arm/boot/dts/rk3128-box.dts
+++ b/arch/arm/boot/dts/rk3128-box.dts
@@ -90,7 +90,8 @@
 		/*0 - Normal, 1 - Force Host, 2 - Force Device*/
 //		rockchip,usb-mode = <1>;
 	};
-	 rockchip_suspend {
+
+	rockchip_suspend: rockchip_suspend {
                 rockchip,ctrbits = <
                         (0
                         |RKPM_CTR_PWR_DMNS
diff --git a/arch/arm/boot/dts/rk3188.dtsi b/arch/arm/boot/dts/rk3188.dtsi
index 74d67bf7a936..e71892813580 100755
--- a/arch/arm/boot/dts/rk3188.dtsi
+++ b/arch/arm/boot/dts/rk3188.dtsi
@@ -501,23 +501,24 @@
 			<&clk_gpu 200000000>, <&aclk_lcdc0 300000000>,
 			<&aclk_lcdc1 300000000>;
 	};
-	rockchip_suspend {     
+
+	rockchip_suspend: rockchip_suspend {
                 //compatible = "rockchip,rkpm_suspend";
                 // define value is in dt-bindint/suspend/rockchip-pm.h
-                rockchip,ctrbits = <	
+                rockchip,ctrbits = <
                 				(
                 				RKPM_CTR_PWR_DMNS
                 				|RKPM_CTR_GTCLKS
                 				|RKPM_CTR_PLLS
                 				|RKPM_CTR_SYSCLK_DIV
                 				)
-                			>;              
+                			>;
               rockchip,pmic-gpios=<
-                                                RKPM_PINGPIO_BITS_OUTPUT(GPIO0_A0,RKPM_GPIO_OUT_L) 
-                                                RKPM_PINGPIO_BITS_INTPUT(GPIO0_A1,RKPM_GPIO_PULL_UP)                           
+                                                RKPM_PINGPIO_BITS_OUTPUT(GPIO0_A0,RKPM_GPIO_OUT_L)
+                                                RKPM_PINGPIO_BITS_INTPUT(GPIO0_A1,RKPM_GPIO_PULL_UP)
                                                 >;
-       
         };
+
 	fb: fb{
 		compatible = "rockchip,rk-fb";
 		rockchip,disp-mode = <DUAL>;
diff --git a/arch/arm/boot/dts/rk3288.dtsi b/arch/arm/boot/dts/rk3288.dtsi
index d3a4563730e5..5fb9529c7666 100755
--- a/arch/arm/boot/dts/rk3288.dtsi
+++ b/arch/arm/boot/dts/rk3288.dtsi
@@ -1272,7 +1272,7 @@
 		interrupt-names = "isp_mmu";
 	};
 
-	rockchip_suspend {
+	rockchip_suspend: rockchip_suspend {
 		rockchip,ctrbits = <
 			(0
 			 |RKPM_CTR_PWR_DMNS
-- 
2.35.3

